pkg/mark/link.go

```diff
@@ -0,0 +1,69 @@
+package mark
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/url"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/kovetskiy/mark/pkg/confluence"
+)
+
+type Link struct {
+	MDLink string
+	Link   string
+}
+
+// ResolveRelativeLinks finds links in the markdown, and replaces one pointing to other Markdowns either by own link (if not created to Confluence yet) or witn actual Confluence link
+func ResolveRelativeLinks(
+	api *confluence.API,
+	markdown []byte,
+	base string,
+) ([]Link, error) {
+	var (
+		links []Link
+	)
+	re := regexp.MustCompile("\\[.*?\\]\\((.*?)\\)")
+	submatchall := re.FindAllStringSubmatch(string(markdown), -1)
+	for _, element := range submatchall {
+		possibleMDFile := element[1]
+		filepath := filepath.Join(base, possibleMDFile)
+		if _, err := os.Stat(filepath); err == nil {
+			linkMarkdown, err := ioutil.ReadFile(filepath)
+			if err != nil {
+				continue
```

Comment:
Errors here and after should be reported to the user. It's not fine that mark can't read a file.

Reply:
Logging errors, but not breaking.